cypress.json

```diff
@@ -5,6 +5,7 @@
     "FULL_INTEGRATION_TEST": false
   },
   "fixturesFolder": "tests/_fixtures",
+  "ignoreTestFiles": "**/tests/_support/utils/__tests__/*",
```

Comment:
Let's not ignore these tests in cypress. Instead, let's have cypress run them!
Find a way around using jasmine.
https://docs.cypress.io/docs/bundled-tools
Use Sinon: http://sinonjs.org/